Parse/PFFile.m

```diff
@@ -315,13 +307,13 @@ - (BFTask *)_uploadFileAsyncWithSessionToken:(NSString *)sessionToken
 #pragma mark Download
 
 - (BFTask *)_getDataAsyncWithProgressBlock:(PFProgressBlock)progressBlock {
-    return [[self _downloadAsyncWithProgressBlock:progressBlock] continueAsyncWithSuccessBlock:^id(BFTask *task) {
+    return [[self _downloadAsyncWithProgressBlock:progressBlock] continueWithSuccessBlock:^id(BFTask *task) {
```

Comment:
No, not really... It was attached to PFTaskQueue vs PFAsyncTaskQueue, which was safer to use with async APIs.
It's no longer needed. `downloadAsync` already performs on the background thread, so there is basically no need to do this scheduling here.